validator/lib/validateTileset.js

```diff
@@ -36,6 +36,13 @@ function validateTopLevel(tileset) {
         return 'Tileset must declare its geometricError as a top-level property.';
     }
 
+    var gltfUpAxis = tileset.asset.gltfUpAxis;
+    if (defined(gltfUpAxis)) {
+        if (gltfUpAxis !== 'X' && gltfUpAxis !== 'Y' && gltfUpAxis !== 'Z') {
+            return 'gltfUpAxis should either be "X", "Y", or "Z".';
+        }
+    }
+  
```

Comment:
Makes sense. Changed the order here and also in tests (to maintain consistency).